docs/cli-reference.md

```diff
@@ -0,0 +1,38 @@
+# Useful commands
+
+Define your **Okteto Desk** using an optional `desk.yml` file. A `desk.yml` file looks like this:
```

Comment:
I'd start with the normal usage, and then have a second section on advanced customization.